indra/newview/llvoavatar.cpp

```diff
@@ -2137,7 +2137,7 @@ void LLVOAvatar::resetSkeleton(bool reset_animations)
     LL_DEBUGS("Avatar") << avString() << " reset starts" << LL_ENDL;
     if (!isControlAvatar() && !mLastProcessedAppearance)
     {
-        LL_WARNS() << "Can't reset avatar; no appearance message has been received yet." << LL_ENDL;
+        LL_WARNS() << "Can't reset avatar '" << getFullname() << "'; no appearance message has been received yet." << LL_ENDL;
```

Comment:
Plese don't use full name here, use agent id. Reasons:
1. Controls avatars have no proper name, they are objects, name is not unique and of little value.
2. Name can change or be requested by uuid
3. Lets not show avatar name in logs an extra time